sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/pipeline/_pipeline_expression.py

```diff
@@ -61,7 +62,7 @@ def _enumerate_operation_combination() -> Dict[str, Union[str, Exception]]:
             for operator in _SUPPORTED_OPERATORS:
                 k = f"{type1} {operator} {type2}"
                 try:
-                    eval_result = eval(f"{operand1} {operator} {operand2}")  # pylint: disable=eval-used
+                    eval_result = literal_eval(f"{operand1} {operator} {operand2}")  # pylint: disable=eval-used
```

Comment:
for `PipelineExpression` scenario, `literal_eval` will not work as expected
please help remain `eval` and add "# nosec" for it here

Reply:
Done.